node/status_node.go

```diff
@@ -107,7 +114,30 @@ func (n *StatusNode) startWithDB(config *params.NodeConfig, db *leveldb.DB, serv
 	}
 
 	if n.discoveryEnabled() {
-		return n.startDiscovery()
+		// Check if LES is enabled before starting Discovery.
+		// If so, add LES topics to required topics.
+		if config.LightEthConfig != nil && config.LightEthConfig.Enabled {
+			var les *les.LightEthereum
+			if err := n.gethService(&les); err != nil {
+				log.Error("failed to get LES service")
+				return err
+			}
+
+			genesisHash := les.BlockChain().Genesis().Hash()
+			les1Topic := discv5.Topic("LES@" + common.Bytes2Hex(genesisHash.Bytes()[0:8]))
+			les2Topic := discv5.Topic("LES2@" + common.Bytes2Hex(genesisHash.Bytes()[0:8]))
+
+			n.config.RequireTopics[les1Topic] = params.Limits{Min: 1, Max: 2}
```

Comment:
i noticed that it is better not to use different limits on end user devices. cause the longer we run discovery the more discovery protocol requests we will get which we will serve and in the end consume more mobile traffic.

Reply:
As so you mean that probably `params.Limits{Min: 1, Max: 1}` or `params.Limits{Min: 2, Max: 2}` would be better?

Reply:
yes